docs/supported-libraries.md

```diff
@@ -19,116 +19,116 @@ These are the supported libraries and frameworks:
+| [Apache Axis2](https://axis.apache.org/axis2/java/core/)                                                                                    | 1.6+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | see Servlet                                                                            |
+| [Apache Camel](https://camel.apache.org/)                                                                                                   | 2.20+ (not including 3.x yet) | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                   |
+| [Apache DBCP](https://commons.apache.org/proper/commons-dbcp/)                                                                              | 2.0+                          | [opentelemetry-apache-dbcp-2.0](../instrumentation/apache-dbcp-2.0/library)                                                                                                                                                                                                                                                                                                             | [Database Pool Metrics]                                                                |
+| [Apache CXF JAX-RS](https://cxf.apache.org/)                                                                                                | 3.2+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                   |
```

Comment:
maybe something like `server span name and http.route`?

Reply:
ya, I like that, since "provides http.route" always means updating server span name for us. we can always have a more detailed explanation of what "provides http.route" means in a footer if needed